workflowproj/workflowproj_test.go

```diff
@@ -76,7 +76,6 @@ func Test_Handler_WorkflowMinimalAndPropsAndSpecAndGeneric(t *testing.T) {
 		WithWorkflow(getWorkflowMinimal()).
 		WithAppProperties(getWorkflowProperties()).
 		AddResource("myopenapi.json", getSpecOpenApi()).
-		AddResource("myopenapi.json", getSpecOpenApi()).
```

Comment:
@ricardozanini was there a reason it was defined twice ?

Reply:
LOL, no